### ISSUE TITLE
Node JS Course: Small rewording of sentence in Controllers lesson, misuse of "Because"

### ISSUE THREAD

**Issue by tgorsk15** [TheOdinProject/curriculum#28516]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The sentence in question is "Because what if we actually want to send a 404?"  In the context of the sentence that comes before it, this does not make sense. Perhaps "Because" should be replaced with the word "But".

![image](https://github.com/user-attachments/assets/6d213987-5b5a-4942-b9ed-97567b4f552f)

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-controllers

### (Optional) Discord Name

tgorsk15

**Comment by MaoShizhong** [comment]
Thanks for this, nice catch.

## Acceptance criteria

- [ ] `Because` is replaced with `But` in the sentence highlighted above.

Please comment if you'd like to be assigned. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by mathdebate09** [comment]
Hey mao, i could fix this up :)

**Comment by MaoShizhong** [comment]
All yours @jomkv
